R/utils.R

```diff
@@ -320,3 +267,12 @@ create_tree_branch <- function(x, title, prefix = "") {
     )
   )
 }
+
+# Renaming of austraits functions to ensure old scripts still work
+
+util_list_to_df1 <- austraits::convert_list_to_df1
```

Comment:
There might be a way to do this that also enables the help for these functions, leveraging the new name. 

I also wonder if we should flag these as deprecated? 

@fontikar What's your advice?

Reply:
Yes, @fontikar and I talked about this, but then ran out of time on Friday. I agree it would be good to print a message when *some* of these are used.

But I think for `database_create_combined_table` we really should just have this name preserved for a while, without any message, since it is shown in the main figure in the publication under that name.

Reply:
Sorry team, for the late replies! 

Best practices are detailed [here ](https://lifecycle.r-lib.org/articles/communicate.html)

I think we need to bring back the old function for now so we can properly implement deprecation. such as `deprecate_soft()` or `deprecate_warn()`

Like this: 
```
add_two <- function(x, y) {
  lifecycle::[deprecate_warn](https://lifecycle.r-lib.org/reference/deprecate_soft.html)("1.0.0", "add_two()", "base::sum()")
  x + y
}
````
Alternatively, we can use `deprecate_stop()` function, forcing the using to update their code to the new function. I am in favour of this given as @dfalster the users of traits.build functions is likely still small-ish where you can be a bit more firm with the deprecation. 

I suggest we follow what is suggested here in [renaming a function](https://lifecycle.r-lib.org/articles/communicate.html#rename-a-function) 

thoughts?

Reply:
I think we should also rexport so we can point to austrait's documentation

```
#' @importFrom magrittr %>%
#' @export
magrittr::`%>%`
```

Reply:
Thanks @fontikar I like your suggestion and commits.